CannibalisticZombies/Assets/Scripts/Inventory.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class Inventory 
+{
+    private List<InventorySlot> inventory;
```

Comment:
Looking deeper into it, I recommend using a Dictionary instead of a list.

`Dictionary<String, Item>`
The String key for the dictionary will be the itemName

[](https://stackoverflow.com/questions/38197707/how-to-find-list-element-with-class-without-looping-unity-c-sharp)
I recommend this because for `AddItem() `and `RemoveItem()` we can easily find if the item is already in the inventory using `dictionaryName.ContainsKey(keyName)`